### PR TITLE
DLS-9575 Upgrade Bootstrap

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -38,6 +38,7 @@ lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
+    onLoadMessage := "",
```
Comment (johno1985) [inline build.sbt:41]: I was expecting to see some additional settings added here for the logging?

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -7,18 +7,19 @@ object AppDependencies {
   val hmrc = "uk.gov.hmrc"
   val hmrcMongo = "uk.gov.hmrc.mongo"
   val mongoVersion = "0.73.0"
+  val bootstrapVersion = "7.8.0"
```
Comment (johno1985) [inline project/AppDependencies.scala:10]: still on 7.8.0

**Review by johno1985 — CHANGES_REQUESTED** [review]
still on version 7.8.0

File: build.sbt (hunk 1 above)
Comment (LAJW) [reply build.sbt:41]: I tried, but it clashes with the flex dependency for some reason.

**Comment by LAJW** [comment]
Upgraded and addressed bot warnings.

**Review by johno1985 — APPROVED** [review]

**Pull request merged**